tsconfig.json

```diff
@@ -1,10 +1,12 @@
 {
   "compilerOptions": {
     "outDir": "lib",
-    "target": "es5",
-    "moduleResolution": "node",
+    "target": "es6",
+    "sourceMap": true,
```

Comment:
with `@babel/preset-typescript`, it's possible to have single `tsconfig.json` file.